addons/l10n_pl/models/account_move.py

```diff
@@ -16,3 +17,11 @@ class AccountMove(models.Model):
         string='B_MPV_Prowizja',
         help="Supply of agency and other services pertaining to the transfer of a single-purpose voucher",
     )
+
+    def action_post(self):
+        "Validation to avoid having credit notes with more than the invoice"
+        for record in self:
+            if record.company_id.account_fiscal_country_id.code == 'PL' and record.reversed_entry_id and\
+                record.reversed_entry_id.amount_total < record.amount_total and record.move_type == 'entry':
```

Comment:
Did you test it ? :thinking:  To me, it should be != 'entry', else it will not trigger on refunds as it should

Reply:
I'm so sorry, I missed this PR with another PR also for PL :dizzy_face: